dotnet/Directory.Packages.props

```diff
@@ -30,11 +30,12 @@
     <PackageVersion Include="Microsoft.Identity.Client" Version="4.61.3" />
     <PackageVersion Include="Microsoft.ML.OnnxRuntime" Version="1.18.0" />
     <PackageVersion Include="FastBertTokenizer" Version="1.0.28" />
+    <PackageVersion Include="Pinecone.NET" Version="2.1.1" />
     <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="8.0.1" />
     <PackageVersion Include="System.Linq.Async" Version="6.0.1" />
     <PackageVersion Include="System.Memory.Data" Version="8.0.0" />
     <PackageVersion Include="System.Numerics.Tensors" Version="8.0.0" />
-    <PackageVersion Include="System.Text.Json" Version="8.0.2" />
+    <PackageVersion Include="System.Text.Json" Version="8.0.3" />
```

Comment:
Unless you need version 8.0.3 for the pinecone code to work, let's not make this change here.  I'm trying to avoid changes unrelated to the memory changes in the feature branch, to make it easier to manage merges for it.

Reply:
Pinecone.NET uses 8.0.3 and going back to 8.0.2 results in package downgrade error in one of the PR checks